backend/audit/excel.py

```diff
@@ -413,17 +408,33 @@ def _extract_data(
     return result
 
 
+def _remove_invalid_award_entries(data):
+    # Filter out invalid entries

```

Comment:
I think it would be good to expand this comment for the future.

What is an `invalid entry`? I'm *assuming* you mean something where the... well, actually, I don't know. The idea of the intake filtering data is something that is worrying, so I thought I'd at least ask.

Reply:
Sorry, I meant to add comments to a few places before publishing this PR. However, as I switched my focus to #1279, I lost track of my thoughts. After adding formulas to auto-calculate federal awards total and cluster total on the sheet, Excel set all the targeted cells associated with these named ranges to 0. So, when uploading federalAwards, Python mistakenly reads extra rows (consisting solely of empty cells but two set to 0) into the JSON object. To resolve this issue, I implemented this method to filter out these unwanted rows (which I am referring to as invalid here). I will update the comment here and possibly rename the method for clarity once I push 1279.